src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2861,6 +2874,15 @@ void Eltwise::prepareParams() {
                                "'");
             }
         }
+        // do output saturation if inputs has constant, this saturation process will be moved to compilation stage in
+        // future
```

Comment:
Totaly not clear why Constants on inputs affects store logic. Need more detailed explanation of the issue.